### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

Data correction for item combo IDs 1573:2716:2971
https://www.divine-pride.net/database/item/1573
https://www.divine-pride.net/database/item/2716
https://www.divine-pride.net/database/item/2971

Data correction for item combo IDs 1618:2509
https://www.divine-pride.net/database/item/1618
https://www.divine-pride.net/database/item/2509

Data correction for item combo IDs 1628:2509
https://www.divine-pride.net/database/item/1628
https://www.divine-pride.net/database/item/2509

Data correction for item combo IDs 1649:2198
https://www.divine-pride.net/database/item/1649
https://www.divine-pride.net/database/item/2198

Data correction for item combo IDs 24084:24085:24086:24087:24088:24089
https://www.divine-pride.net/database/item/24084
https://www.divine-pride.net/database/item/24085
https://www.divine-pride.net/database/item/24086
https://www.divine-pride.net/database/item/24087
https://www.divine-pride.net/database/item/24088
https://www.divine-pride.net/database/item/24089

Data correction for item IDs 1498, 1580, 1597, 1616, 1623, 1627, 1628, 1629, 1638, 1660, 1673, 1674, 1676, 1678, 1680, 1682, 1683, 1684, 1685, 1686, 1687, 1690, 1691, 1692, 1693, 1694, 1695, 1696, 1697, 1698, 1736, 1737, 1839, 1939, 1995, 2001, 2002, 2003, 2006, 2025, 13454, 16040, 18130, 21015, 24527, 24587, 24656, 28007, 28106, 28604, 28705

https://www.divine-pride.net/database/item/1498
https://www.divine-pride.net/database/item/1580
https://www.divine-pride.net/database/item/1597
https://www.divine-pride.net/database/item/1616
https://www.divine-pride.net/database/item/1623
https://www.divine-pride.net/database/item/1627
https://www.divine-pride.net/database/item/1628
https://www.divine-pride.net/database/item/1629
https://www.divine-pride.net/database/item/1638
https://www.divine-pride.net/database/item/1660
https://www.divine-pride.net/database/item/1673
https://www.divine-pride.net/database/item/1674
https://www.divine-pride.net/database/item/1676
https://www.divine-pride.net/database/item/1678
https://www.divine-pride.net/database/item/1680
https://www.divine-pride.net/database/item/1682
https://www.divine-pride.net/database/item/1683
https://www.divine-pride.net/database/item/1684
https://www.divine-pride.net/database/item/1685
https://www.divine-pride.net/database/item/1686
https://www.divine-pride.net/database/item/1687
https://www.divine-pride.net/database/item/1690
https://www.divine-pride.net/database/item/1691
https://www.divine-pride.net/database/item/1692
https://www.divine-pride.net/database/item/1693
https://www.divine-pride.net/database/item/1694
https://www.divine-pride.net/database/item/1695
https://www.divine-pride.net/database/item/1696
https://www.divine-pride.net/database/item/1697
https://www.divine-pride.net/database/item/1698
https://www.divine-pride.net/database/item/1736
https://www.divine-pride.net/database/item/1737
https://www.divine-pride.net/database/item/1839
https://www.divine-pride.net/database/item/1939
https://www.divine-pride.net/database/item/1995
https://www.divine-pride.net/database/item/2001
https://www.divine-pride.net/database/item/2002
https://www.divine-pride.net/database/item/2003
https://www.divine-pride.net/database/item/2006
https://www.divine-pride.net/database/item/2025
https://www.divine-pride.net/database/item/13454
https://www.divine-pride.net/database/item/16040
https://www.divine-pride.net/database/item/18130
https://www.divine-pride.net/database/item/21015
https://www.divine-pride.net/database/item/24527
https://www.divine-pride.net/database/item/24587
https://www.divine-pride.net/database/item/24656
https://www.divine-pride.net/database/item/28007
https://www.divine-pride.net/database/item/28106
https://www.divine-pride.net/database/item/28604
https://www.divine-pride.net/database/item/28705

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -19603,7 +19605,6 @@ Body:
     Script: |
       bonus bAtkEle,Ele_Holy;
       bonus bDex,4;
-      bonus bMatkRate,15;
       bonus2 bSubRace,RC_Demon,15;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:19608]: Aegis item script give matkrate +15%
```suggestion
      bonus bMatkRate,15;
      bonus2 bSubRace,RC_Demon,15;
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -19646,10 +19647,9 @@ Body:
       bonus2 bIgnoreMdefRaceRate,RC_Player_Human,25;
       bonus bUnbreakableWeapon;
       if (.@r>5) {
+         bonus bMatkRate,5;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:19650]: We shouldn't change the script the description is too ambiguous, but we may update it with the aegis data.
```
      .@r = min(14,getrefine());
      bonus bMatkRate,.@r;
      bonus bUnbreakableWeapon;
      if (.@r>5) {
         .@bonus1 = 5;
         .@bonus2 = (.@r-5) * 2;
      }
      if (.@r>8) {
         bonus5 bAutoSpellOnSkill,"WZ_STORMGUST","MG_SAFETYWALL",10,200,1;
         bonus5 bAutoSpellOnSkill,"WZ_METEOR","MG_SAFETYWALL",10,200,1;
         bonus5 bAutoSpellOnSkill,"WZ_VERMILION","MG_SAFETYWALL",10,200,1;
      }
      bonus2 bMagicAddRace,RC_DemiHuman,15+.@bonus2;
      bonus2 bMagicAddRace,RC_Player_Human,15+.@bonus2;
      bonus2 bIgnoreMdefRaceRate,RC_DemiHuman,25+.@bonus1+.@bonus2;
      bonus2 bIgnoreMdefRaceRate,RC_Player_Human,25+.@bonus1+.@bonus2;
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -105441,6 +105443,7 @@ Body:
       NoAuction: true
     Script: |
       bonus bMaxHP,100;
+      bonus bMatk,15;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:105446]: The matk is already written in the MagicAttack field
```suggestion
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:19650]: Thanks

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**